File: server/当前版本/server/zlj1/game_server/src/com/imop/lj/gameserver/item/template/LeaderSkillBookTemplateVO.java
package com.imop.lj.gameserver.item.template;

import com.imop.lj.core.annotation.ExcelRowBinding;
import com.imop.lj.common.exception.TemplateConfigException;
import com.imop.lj.core.annotation.ExcelCellBinding;

/**
 * 人物技能书
 * 
 * @author CodeGenerator, don't modify this file please.
 */

@ExcelRowBinding
public abstract class LeaderSkillBookTemplateVO extends ItemTemplate {

	/** 职业要求 */
	@ExcelCellBinding(offset = 34)
	protected int jobLimit;

	/** 技能ID */
	@ExcelCellBinding(offset = 35)
	protected int skillId;


	public int getJobLimit() {
		return this.jobLimit;
	}

	public void setJobLimit(int jobLimit) {
		if (jobLimit < 1) {
			throw new TemplateConfigException(this.getSheetName(), this.getId(),
					35, "[职业要求]jobLimit的值不得小于1");
		}
		this.jobLimit = jobLimit;
	}
	
	public int getSkillId() {
		return this.skillId;
	}

	public void setSkillId(int skillId) {
		this.skillId = skillId;
	}
	

	@Override
	public String toString() {
		return "LeaderSkillBookTemplateVO[jobLimit=" + jobLimit + ",skillId=" + skillId + ",]";

	}
}

File: u3u2/Assets/Scripts/app/db/AutoCfg/LeaderSkillBookTemplateVO.cs
using System;
using System.Collections.Generic;
using UnityEngine;

namespace app.db
{
	/**
	 * 人物技能书
	 * 
	 * @author CodeGenerator, don't modify this file please.
	 */
	public abstract class LeaderSkillBookTemplateVO : ItemTemplate
	{
	/// <summary>
    /// 职业要求
    /// </summary>
	//@ExcelCellBinding(offset = 38)
	public int jobLimit;

	/// <summary>
    /// 技能ID
    /// </summary>
	//@ExcelCellBinding(offset = 39)
	public int skillId;


}
}